osu.Game/Screens/Multi/Header.cs

```diff
@@ -71,11 +72,20 @@ private void load(OsuColour colours)
 
         private class MultiHeaderTitle : ScreenTitle
         {
+            private const int text_offset = 2;
```

Comment:
Single-use constant (can be inlined), and again, not sure why this is required.

Reply:
This was copied over from the previous version of `ScreenTitle`, it was used there to align the text to match the desired design I suppose? Can remove if not necessary anymore.

Reply:
If nobody knows why it's there let's just remove it.